src/main/kotlin/com/github/mitchellolsthoorn/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -2,75 +2,194 @@ package com.github.mitchellolsthoorn.testgenie.toolwindow
 
 import com.intellij.openapi.ui.Messages
 import com.intellij.openapi.wm.ToolWindow
-import java.awt.Dimension
 import java.awt.event.ActionEvent
 import javax.swing.JButton
-import javax.swing.JComboBox
+import com.intellij.openapi.ui.ComboBox
+import com.intellij.ui.components.JBLabel
+import com.intellij.ui.components.JBScrollPane
+import com.intellij.util.ui.FormBuilder
+import java.awt.*
 import javax.swing.JComponent
 import javax.swing.JLabel
 import javax.swing.JPanel
-import javax.swing.JTextField
+import javax.swing.JSpinner
+import javax.swing.SpinnerNumberModel
 
 /**
  * This class stores the UI of the TestGenie tool window.
  */
 class TestGenieToolWindow(_toolWindow: ToolWindow) {
 
-    private val toolWindow = _toolWindow
+    private val toolWindow : ToolWindow = _toolWindow
```

Comment:
are we refactoring (removing) this later?

Reply:
I think that @jzelenjak said that this is needed for a something in other issue branch. I am not sure if we will remove it later or what exactly it is.

Reply:
Yes, we cannot simply remove it here as it would require us to also change the call in toolwinowfactory class, which is also used by Marti. I didn't want to introduce problems for him